packages/compass-preferences-model/src/preferences.ts

```diff
@@ -635,25 +631,6 @@ export const storedUserPreferencesProps: Required<{
     validator: z.boolean().default(false),
     type: 'boolean',
   },
-  /**
-   * Chooses atlas service backend configuration from preset
-   *  - compass-dev: locally running compass kanopy backend (localhost)
-   *  - compass:    compass kanopy backend (compass.mongodb.com)
-   *  - atlas-dev:  dev mms backend (cloud-dev.mongodb.com)
-   *  - atlas:      mms backend (cloud.mongodb.com)
-   */
-  atlasServiceConfigPreset: {
```

Comment:
@gribnoysup was this preference intended to be kept in?

Reply:
This was meant to stay yeah, also definitely a bit premature to do this operation

Reply:
Chatting on slack, we're thinking let's keep the preference, but rename it and default it to `atlas` so that it's consistent for everyone.